eventboss.gemspec

```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.13"
   spec.add_development_dependency 'rake', '>= 10.0'
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rails", "~> 5.0"
```

Comment:
Isn't that an overkill?

Reply:
How can we test this new handler without it?

Reply:
Defining a mock/stub in the test?